src/shared-components/verificationMessages/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { TransitionGroup } from 'react-transition-group';
+
+import { CenteredMessageList, MessageList, MessageItem } from './style';
+import HelperTransition from '../../utils/helperTransition';
+import formatMessage from './formatMessage';
+
+export type messagesTypes = 'error' | 'success';
```

Comment:
[Nit] We should rename these MessageTypes and MessageType so it's clear just from looking at the name that these are types and not actual values.

Reply:
fixed